java/ql/lib/semmle/code/java/frameworks/IoJsonWebToken.qll

```diff
@@ -0,0 +1,11 @@
+/** Predicates and classes to reason about the `io.jsonwebtoken` library. */
+
+import java
+import semmle.code.java.dataflow.DataFlow
+import semmle.code.java.dataflow.FlowSteps
+
+private class JwsHeaderFieldsInheritTaint extends DataFlow::SyntheticFieldContent,
+  TaintInheritingContent
+{
+  JwsHeaderFieldsInheritTaint() { this.getField().matches("io.jsonwebtoken.JwsHeader.%") }
```

Comment:
I don't expect this to work, if this file is not bidirectionally imported.

Reply:
Correct, it had caused some problems. I added an import to `FlowSteps::Frameworks` and it seems to work now.